### ISSUE TITLE
Bad platform detection for IE Edge

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/phet-core#20]
It looks like unit tests in https://github.com/phetsims/tasks/issues/98#issuecomment-112210664 are trying to be run on IE when they are supposed to be FF/Chromium only:
```js
  // We can only guarantee comparisons for Firefox and Chrome
  if ( !phetCore.platform.firefox && !phetCore.platform.chromium ) {
    window.console && window.console.log && window.console.log( 'Not running pixel-comparison tests' );
    return;
  }
```

We'll want to re-examine the platform detection here.

**Comment by jonathanolson** [comment]
According to https://msdn.microsoft.com/en-us/library/hh869301(v=vs.85).aspx, the user-agent will be like:

Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.71 Safari/537.36 Edge/12.0

**Comment by jonathanolson** [comment]
This is relevant for:
Build a Molecule (.cur vs .png for cursors, will need to test)
Joist (check IE11/Edge compatibility for full-screen)
Scenery (pixel-comparison tests)

**Comment by jonathanolson** [comment]
We should check for navigator.appName also.

**Comment by pixelzoom** [comment]
@jonathanolson Can you summarize here for those of us who are interested in how to detect Edge? I looked at the commit, and it's not obvious (to me anyway) what you ended up doing.

**Comment by jonathanolson** [comment]
Still just looking at the user-agent, detection of "Edge/" is used, and it won't report as platform.chromium anymore. I checked through recent browsers to make sure each one only returned true for its specific platform/version.

Also, Edge returns "Netscape" for navigator.appName. So, yeah.

Let me know if I should add documentation.

**Comment by pixelzoom** [comment]
Yeah, documentation (similar to what you said in the preceding comment) would be probably be a good idea.

**Comment by jonathanolson** [comment]
Let me know if this is sufficient.
